client/tokenized-express-checkout/index.js

```diff
@@ -374,8 +389,9 @@ jQuery( ( $ ) => {
 						}
 
 						expressCheckoutButtonUi.unblockButton();
+						expressCheckoutButtonUi.showContainer();
 					} catch ( e ) {
-						expressCheckoutButtonUi.hide();
```

Comment:
The `hide()` method doesn't exist in `expressCheckoutButtonUi`. Fixed.

Reply:
Whoops, not sure what's better – the fact that you fixed it or that we seem to have never hit the exception handling in this code (I assume so, since we haven't encountered this before) 😆

Anyway, thanks for spotting and fixing this 💪